### PR TITLE
web-angular: upgrade ACX to +15

### PR DESCRIPTION
**Pull request by chalin**

### CODE REVIEW

File: templates/web-angular/pubspec.yaml [hunk 1]
```diff
@@ -9,8 +9,7 @@ environment:
 
 dependencies:
   angular: ^5.0.0-alpha+14
```
Comment (kwalrath) [inline templates/web-angular/pubspec.yaml]: not +15?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,9 @@
 
 - `web-angular` template:
   - Set min SDK version to 2.0.0-dev.60.0.
-  - Updated dependencies to include `-alpha+15`.
+  - Updated dependencies to include `-alpha+15` versions
+    of `angular` and `angular_components`
+  - Temporarily override `build_config` with `^0.3.0`
```
Comment (kwalrath) [inline CHANGELOG.md]: Add periods to the end of list items.

File: templates/web-angular/pubspec.yaml (hunk 1 above)
Comment (chalin) [reply templates/web-angular/pubspec.yaml]: Oops, yes (the +15 was in the override that I removed :-/ ... :). Done.

File: CHANGELOG.md (hunk 2 above)
Comment (chalin) [reply CHANGELOG.md]: Done

**Pull request merged**